python/ray/dag/tests/experimental/test_accelerated_dag.py

```diff
@@ -1424,6 +1462,24 @@ def get_node_id(self):
     compiled_dag.teardown()
 
 
+@ray.remote
+class TestWorker:
```

Comment:
Had to move this out otherwise a weird error happens:

```
E                       ray::Replica.__init__() (pid=19780, ip=127.0.0.1, actor_id=0dee9f2dbd6be769cf7cf52d01000000, repr=<test_accelerated_dag.FunctionActorManager._create_fake_actor_class.<locals>.TemporaryActor object at 0x109b04880>)
E                       ModuleNotFoundError: No module named 'test_accelerated_dag'
```